src/main/java/com/exactpro/th2/readlog/RegexLogParser.java

```diff
@@ -15,20 +15,37 @@
  */
 package com.exactpro.th2.readlog;
 
+import java.io.StringWriter;
 import java.time.DateTimeException;
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Comparator;
```

Comment:
import java.util.Comparator;
import javax.annotation.Nullable;
import org.apache.commons.text.lookup.StringLookupFactory;

Can be removed